### DISCUSSION TITLE
Feature: Remove close Button (X) at center top

### DISCUSSION THREAD

**Discussion by ghost** [ChristopherKlay/StadiaEnhanced#237] [Ideas & Requests]
Feature request: Is it possible for you to remove the annoying close button at the top of the screen that only appears when you move the mouse to the top of your screen? 

Use case: Start a game (for example Doom) where the visual cursor disappears when you are in game and move your mouse to the top. You will see that this close button (X) will always appear.

**Comment by ChristopherKlay** [comment]
I'll see how well supported styling/hiding said element is across some popular browsers and note it for a future feature (:
